### PR TITLE
Add python function and cli to download qsiprep qc metrics and perform exclusion

### PR DESCRIPTION
**Pull request by richford**

### CODE REVIEW

**Comment by richford** [comment]
@arokem, @jyeatman. This is the QC function that we discussed on the call today.

File: notebooks/qsiprep_qc_dataframe.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    masks = {}
+    for qc_metric, (p_lo, p_hi) in exclusion_criteria.items():
+        masks[qc_metric] = np.logical_and(
+            df_qc[qc_metric] >= df_qc[qc_metric].quantile(p_lo),
+            df_qc[qc_metric] <= df_qc[qc_metric].quantile(p_hi)
+        )
+        
+        df_qc[qc_metric + "_pass"] = masks[qc_metric]
+    
+    union = functools.reduce(operator.mul, list(masks.values()), 1)
```
Comment (arokem) [inline notebooks/qsiprep_qc_dataframe.py:43]: Nice one

**Review by arokem — APPROVED** [review]
Looks great! Did you by chance run this and save the output somewhere on S3 where Ethan can grab it for his analysis?

**Comment by richford** [comment]
[s3://hbn-afq/derivatives/afq/dwiqc.csv](https://hbn-afq.s3-us-west-2.amazonaws.com/derivatives/afq/dwiqc.csv). I'll post it to slack as well.

**Pull request merged**